src/main/scala/com/datastax/gatling/plugin/response/DseResponseHandler.scala

```diff
@@ -153,8 +155,26 @@ class CqlResponseHandler[T <: Statement[T], B <: StatementBuilder[B,T]](val next
                          val metricsLogger: MetricsLogger)
   extends DseResponseHandler[T, AsyncResultSet, CqlResponse] {
   override protected def tag: String = dseAttributes.tag
-  override protected def queries: Seq[String] = Seq.empty
+  override protected def queries: Seq[String] = formatStatement(stmt)
   override protected def specificChecks: List[DseCqlCheck] = dseAttributes.cqlChecks
   override protected def newResponse(rs: AsyncResultSet): CqlResponse = new CqlResponse(rs, dseAttributes)
   override protected def coordinator(rs: AsyncResultSet): Node = rs.getExecutionInfo.getCoordinator
+
+  def formatSimpleStatement(s:SimpleStatement):String = s.getQuery
+
+  def formatBoundStatement(s:BoundStatement):String = s.getPreparedStatement.getQuery
+
+  def formatStatement(stmt:Statement[T]):Seq[String] = {
+
+    stmt match {
+      case s:SimpleStatement => Seq(formatSimpleStatement(s))
```

Comment:
nit: I think you can create a method with a type suffix: 
`def formatStatement(s:BoundStatement):String`
`def formatStatement(s:SimpleStatement)`
They should not clash because `BoundStatement` is not extending `SimpleStatement` isn't it?

Reply:
The problem with that is formatStatement(stmt:Statement). Because both BoundStatement and SimpleStatement inherit from Statement things would get ambiguous.  Also note that formatStatement() returns a Seq[String] while the fns for the individual types exist only to extract values out of instances of those types.

All that said, I think I can probably change these names to something saner... I'll see if I can clean this up.